cluster/src/main/java/com/hartwig/pipeline/calling/sage/SageSomaticCaller.java

```diff
@@ -0,0 +1,96 @@
+package com.hartwig.pipeline.calling.sage;
+
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import com.hartwig.pipeline.Arguments;
+import com.hartwig.pipeline.ResultsDirectory;
+import com.hartwig.pipeline.alignment.AlignmentPair;
+import com.hartwig.pipeline.datatypes.DataType;
+import com.hartwig.pipeline.datatypes.FileTypes;
+import com.hartwig.pipeline.execution.PipelineStatus;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.BashStartupScript;
+import com.hartwig.pipeline.execution.vm.VirtualMachineJobDefinition;
+import com.hartwig.pipeline.execution.vm.VmDirectories;
+import com.hartwig.pipeline.execution.vm.unix.UnzipToDirectoryCommand;
+import com.hartwig.pipeline.metadata.SomaticRunMetadata;
+import com.hartwig.pipeline.reruns.PersistedDataset;
+import com.hartwig.pipeline.resource.ResourceFiles;
+import com.hartwig.pipeline.stages.SubStageInputOutput;
+import com.hartwig.pipeline.storage.RuntimeBucket;
+
+public class SageSomaticCaller extends SageCaller {
+
+    public static final String NAMESPACE = "sage_somatic";
```

Comment:
What impact will this have downstream on reporting, etc? Will we need to do another rerun to make data homogenous again? 

We could limit this impact of this change by not changing this namespace yet and making that a separate ticket.

Reply:
Done. Reverted to sage.